test.js

```diff
@@ -14,3 +14,10 @@ test('--regex', async t => {
 	await execa('./cli.js', ['--regex=\\bb.*?\\b', '--replacement=foo', filePath]);
 	t.is(fs.readFileSync(filePath, 'utf8'), 'foo foo foo');
 });
+
+test('test new line', async t => {
+	const filePath = await tempWrite(',');
+	await execa('./cli.js', ['--string=,', '--replacement=\n', filePath]);
+	t.is(fs.readFileSync(filePath, 'utf8'), '\n');
```

Comment:
Maybe we should be testing for \r and \t too? To do so, you can duplicate these two lines but with a different replacement.

Reply:
Edited the test to test \r and \t too.